### PR TITLE
Fix absolute path to src

### PR DESCRIPTION
**Pull request by phillipskevin**
Fixing https://github.com/donejs/donejs/issues/353.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
```
Comment (matthewp) [inline .travis.yml]: Are we dropping support for these? Is that documented in an issue somewhere?

File: package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
   "main": "lib",
   "scripts": {
     "coverage": "istanbul cover _mocha -- test/ --timeout 600000",
-    "test": "mocha test/ --timeout 600000",
+    "test": "mocha test/app.test.js --timeout 600000",
```
Comment (matthewp) [inline package.json]: I assume this is a mistake from debugging, we want to run the other tests too.

**Comment by phillipskevin** [comment]
@Sinjhin maybe it would be easier to make a new PR with just the changes to `app/index.js` and `test/app.test.js`?

File: .travis.yml (hunk 1 above)
Comment (Sinjhin) [inline .travis.yml]: https://bitovi.slack.com/archives/team-open-source/p1477096789007575

Comment (matthewp) [inline .travis.yml]: I disagree with that comment, we can't drop compatible on a patch release.  cc @justinbmeyer

Comment (phillipskevin) [inline .travis.yml]: we are going to close this PR and release this change with the 1.0 release.

**Comment by phillipskevin** [comment]
This is not going to be release in `0.9.x`.

File: .travis.yml (hunk 1 above)
Comment (matthewp) [inline .travis.yml]: That's fine, we should still have an issue to discuss 1.0 compatibility though.

Comment (phillipskevin) [inline .travis.yml]: Here is the issue: https://github.com/donejs/donejs/issues/705.

**Pull request closed without merging**